vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -192,10 +192,8 @@ $global:map = @{
         "https://acs-mirror.azureedge.net/kubernetes/v1.23.7/windowszip/v1.23.7-1int.zip",
         "https://acs-mirror.azureedge.net/kubernetes/v1.23.8-hotfix.20220728/windowszip/v1.23.8-hotfix.20220728-1int.zip",
         "https://acs-mirror.azureedge.net/kubernetes/v1.23.12-hotfix.20220922/windowszip/v1.23.12-hotfix.20220922-1int.zip", 
-        "https://acs-mirror.azureedge.net/kubernetes/v1.24.0/windowszip/v1.24.0-1int.zip",
-        "https://acs-mirror.azureedge.net/kubernetes/v1.24.3/windowszip/v1.24.3-1int.zip",
-        "https://acs-mirror.azureedge.net/kubernetes/v1.24.6/windowszip/v1.24.6-1int.zip",
-        "https://acs-mirror.azureedge.net/kubernetes/v1.25.2/windowszip/v1.25.2-1int.zip"
```

Comment:
k8s 1.25.2 will be previewed this week, so we would need it on the VHD for Windows

Reply:
@ganeshkumar5699, v1.25.2 is not available so I only add v1.24 in this PR